scripts/run_model.py

```diff
@@ -1,63 +1,18 @@
 import logging
-import os
-import shutil
 import sys
 
-# These have to be before we do any import from keras.  It would be nice to be able to pass in a
-# value for this, but that makes argument passing a whole lot more complicated.  If/when we change
-# how arguments work (e.g., loading a file), then we can think about setting this as a parameter.
-import random
-import numpy
-random.seed(13370)
-numpy.random.seed(1337)
-
-# pylint: disable=wrong-import-position
-
-# Unless I find a better way to do this, we'll use ConfigFactory to read in a parameter file.
-# ConfigFactory.parse_file() returns a ConfigTree, which is a subclass of OrderedDict, so the
-# return value actually matches the type our code expects, anyway.
-import pyhocon
-
-sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
-from deep_qa.common.checks import ensure_pythonhashseed_set, log_keras_version_info
-from deep_qa.common.params import Params, replace_none
-from deep_qa.common.tee_logger import TeeLogger
-from deep_qa.models import concrete_models
-from keras import backend as K
+from deep_qa.common.checks import ensure_pythonhashseed_set
+from deep_qa.training import run_model
```

Comment:
This is actually going to import keras, before `prepare_environment` is called, because you're running `deep_qa/training/__init__.py`, which imports `Trainer`, which imports keras.  I guess it's not just for cosmetic reasons that we need to move this to `deep_qa/run.py` - we _have_ to in order to get the import order right.